expath-pkg.xml.tmpl

```diff
@@ -3,4 +3,5 @@
     <title>eXist-db Public Application Repository</title>
     <dependency processor="http://exist-db.org" semver-min="3.5.0"/>
     <dependency package="http://expath.org/ns/crypto" semver-min="0.5"/>
+    <dependency package="http://exist-db.org/xquery/semver" semver-min="2.2.0-SNAPSHOT"/>
```

Comment:
Since `SNAPSHOTS` are moving targets wouldn't a full release here better

Reply:
Yes. I will fix this.